templates/issue.html

```diff
@@ -292,41 +300,29 @@
 					<input name="issue_number" value="{{.IssueNumber}}" hidden></input>
 					<input name="repo_id" value="{{.RepoId}}" hidden></input>
```

Comment:
repo Id should be derived from get current repo instead of passed from `<form>`.

Reply:
done